### PR TITLE
Feature/ac 24629 | Path instruction to Create new “Meeting Outcome” picklist field | AC Package 1.5

### PR DESCRIPTION
**Pull request by AmarenderN**
Depends on [224](https://github.com/advisors-excel-llc/sf-ac/pull/224)

### CODE REVIEW

**Review by jduclos-ae — CHANGES_REQUESTED** [review]
Will test section a once the other PR is ready, aside from the few typos, looks like the rest are running ok.

File: 1.5.1 [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        },
+        {
+            "sObjectName": "Activity",
+            "instructionType": "Modify",
+            "instructionTypeName": "FieldLevelSecurity",
+            "taskList": [
+                {
+                    "jiraTicket": "AC-1872a",
+                    "actionType": "FieldLevelSecurity",
+                    "actionTypeName": "command__Meeting_Outcomer__c",
```
[the remark this answers is not in the document]
Comment (AmarenderN) [reply 1.5.1:83]: This is Not ready forgot to  remove "Ready" the label, as it is depends on the rank related work.I will push updated instruction here with the working one

File: 1.5.1 [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
```
[the remark this answers is not in the document]
Comment (AmarenderN) [reply 1.5.1]: Corrected the API Name,But we can test this after the rank related sf-ac pr

File: 1.5.1 [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
```
[the remark this answers is not in the document]
Comment (AmarenderN) [reply 1.5.1]: Corrected the API Name,

**Review by jduclos-ae — APPROVED** [review]

**Pull request merged**